package.json

```diff
@@ -1082,10 +1082,10 @@
     "@mongodb-js/connection-form": "^1.22.2",
     "@mongodb-js/connection-info": "^0.1.2",
     "@mongodb-js/mongodb-constants": "^0.10.2",
-    "@mongosh/browser-runtime-electron": "^2.1.1",
-    "@mongosh/i18n": "^2.1.1",
-    "@mongosh/service-provider-server": "^2.1.1",
-    "@mongosh/shell-api": "^2.1.1",
+    "@mongosh/browser-runtime-electron": "^2.2.10",
+    "@mongosh/i18n": "^2.2.10",
+    "@mongosh/service-provider-server": "^2.2.10",
+    "@mongosh/shell-api": "^2.2.10",
```

Comment:
Should this bump the driver and `mongodb-data-service` as well? Typically, our release flow is:

- Driver package(s) is released
- (If necessary) Bump driver in oidc-plugin
- (If necessary) Bump driver, oidc-plugin in devtools-connect
- Bump driver, oidc-plugin, devtools-connect in mongosh
- Bump driver, oidc-plugin, devtools-connect, mongosh in Compass packages (e.g. data-service)
- Bump driver, oidc-plugin, devtools-connect, mongosh, data-service in VSCode

so that all packages in the source tree are on consistent versions of the driver

Reply:
add these deps were updated in https://github.com/mongodb-js/vscode/pull/734 and only mongosh were left out (probably due to failing ts).

Reply:
Yes https://github.com/mongodb-js/vscode/pull/734 is merged, so main should have those bumped by now.